src/validationinterface.h

```diff
@@ -86,7 +87,11 @@ class CValidationInterface {
      *
      * Called on a background thread.
      */
-    virtual void UpdatedBlockTip(const CBlockIndex *pindexNew, const CBlockIndex *pindexFork, bool fInitialDownload) {}
+    virtual void UpdatedBlockTip(
+        const CChainState* chainstate,
```

Comment:
In commit "validationinterface: add unused CChainState parameter" (0b33ff5bcba7f2aa357a1a12fe1e1452c23defb7)

Will these new chainstate parameters ever be null? Is any implementation ever going to check that they are not null before using them? Would suggest changing to `const CChainState& chainstate` to avoid mishaps otherwise.